### PR TITLE
IP-3668: Update Vm affinity rule implementation to use new endpoints

### PR DESCRIPTION
**Pull request by fradinni**

### CODE REVIEW

File: src/sdk/model/org/org.ts [hunk 1]
```diff
@@ -1462,32 +1459,6 @@ export class Org extends Entity {
     });
   }
 
-  /**
-   * Gets the org's current DRS affinity rules.
-   * @returns {Promise<Array<VmAffinityRule>>} a promise that resolves with the list of DRS affinity rules
-   */
-  /* istanbul ignore next: autogenerated */
-  async getDrsAffinityRules(): Promise<Array<VmAffinityRule>> {
```
Comment (cfsnyder) [inline src/sdk/model/org/org.ts]: i thought this one was still going to be available in the api?

File: src/sdk/model/vdc/vm-affinity-rule-list.ts [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import { VmAffinityRuleJson } from './__json__/vm-affinity-rule-json';
+import { VmAffinityRuleListJson } from './__json__/vm-affinity-rule-list-json';
+
+/* istanbul ignore next: autogenerated */
+export class VmAffinityRuleList {
```
Comment (cfsnyder) [inline src/sdk/model/vdc/vm-affinity-rule-list.ts]: where is this used?

File: src/sdk/model/vdc/__json__/vm-affinity-rule-list-json.ts [hunk 3]
```diff
@@ -0,0 +1,5 @@
+import { VmAffinityRuleJson } from './vm-affinity-rule-json';
+
+export interface VmAffinityRuleListJson {
```
Comment (cfsnyder) [inline src/sdk/model/vdc/__json__/vm-affinity-rule-list-json.ts]: this still doesn't appear to be used anywhere?

Comment (fradinni) [reply src/sdk/model/vdc/__json__/vm-affinity-rule-list-json.ts]: Oh i remove the class but not the JSON descriptor... doing it now.

Comment (fradinni) [reply src/sdk/model/vdc/__json__/vm-affinity-rule-list-json.ts]: Done.

**Pull request merged**